src/Senders/Jaeger.Senders.Grpc/protos/collector.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
Are these straight copies from jaeger-idl, or modified somehow? If not modified, any reason not to use the git submodule as the other repos are doing? If modified, is it documented somewhere _how_ they are modified?

Reply:
Modified to remove the gogoproto stuff to allow Grpc.Tools to generate the cs files from proto. But I just saw that jaeger-idl is using docker and also generating csharp. Will use the generated ones then like with Thrift.

Reply:
This is currently blocked by https://github.com/jaegertracing/jaeger-idl/issues/62

Reply:
I locally generate them through jaeger-idl (from PR) and added the generated code to the class as `Jaeger.Communication.Grpc`.

Reply:
does it mean what we can remove proto files from this PR?

Reply:
Yes, they are removed in latest commits. Depending on if we are able to update jaeger-protobuf and jaeger-idl, I will either keep pre-generated code from idl in here or include it as submodule. Submodule would need us to also add more submodules to all referenced protos like gogo and the api annotations.

Reply:
This is still blocked by:
* https://github.com/jaegertracing/jaeger-idl/pull/63
* https://github.com/jaegertracing/docker-protobuf/pull/15

I don't see them to move fast. So I will use the pre-generated files for now to close this. Regenerating the protos will be painful until those are resolved.